projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -247,6 +248,32 @@ public static function style( $style ) {
 	public static function style_on() {
 		return self::style( true );
 	}
+	/**
+	 * Adds a quick link to the admin bar for the contact form entries.
+	 *
+	 * @param \WP_Admin_Bar $admin_bar The admin bar object.
+	 */
+	public static function add_quick_link_to_admin_bar( \WP_Admin_Bar $admin_bar ) {
+
+		if ( ! current_user_can( 'edit_pages' ) ) {
+			return;
+		}
+
+		$url = ( new Dashboard_View_Switch() )->get_forms_admin_url();
+
+		$admin_bar->add_menu(
+			array(
+				'id'     => 'jetpack-forms',
+				'parent' => null,
+				'group'  => null,
+				'title'  => __( 'Form Responses', 'jetpack-forms' ),
```

Comment:
Sentence case; Form responses.

Reply:
In most cases such as "Edit Page" or "Edit Post" we don't see an sentence case. So I think keeping it the way it is works.

Reply:
I was thinking older convo where Joen and Jay said it's fine to be inconsistent temporarily (or we'll never be able to migrate to sentence case), but they mentioned wp-admin so far favoring Title Case so it's fine: 
- https://github.com/Automattic/jetpack/pull/41086#issuecomment-2593257138